hooks/useFileUpload.ts

```
@@ -1,8 +1,8 @@
-import { MAX_FILE_SIZE, ONE_MB } from "@/lib/consts";
 import { uploadFile } from "@/lib/arweave/uploadFile";
 import { Dispatch, SetStateAction, useState } from "react";
 import captureImageFromVideo from "@/lib/captureImageFromVideo";
 import base64ToFile from "@/lib/base64ToFile";
+import clientUploadToArweave from "@/lib/arweave/clientUploadToArweave";
```

ESLint

[error] 'clientUploadToArweave' is defined but never used.

(@typescript-eslint/no-unused-vars)